### PR TITLE
Added Santiago de Compostela

### LINKED ISSUE #1877: Add Santiago de Compostela
[Santiago_de_Compostel1.docx](https://github.com/slurpcode/slurp/files/9689649/Santiago_de_Compostel1.docx)

https://slurp.readthedocs.io/en/latest/index.html

Author is Idais

### PR DESCRIPTION
**Pull request by vihangamarasinghe**
fixes #1877 ,please review

### CODE REVIEW

**Comment by island-coder** [comment]
> File name is wrong. Extra `.`
> 
> Screen Shot 2022-10-06 at 5 53 02 pm

sorry,fixed it

File: doc/source/santiago_de_compostela.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+﻿# 🇪🇸 Santiago de Compostela
```
Comment (jbampton) [inline doc/source/santiago_de_compostela.md:1]:
```suggestion
﻿# 🇪🇸 Santiago de Compostela
```

File: doc/source/santiago_de_compostela.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+We have made a list of the essential places that you must visit if you go to Santiago de Compostela, so that you do not miss the most important things.
+
+## The Cathedral of Santiago of Compostela 
+
+The Cathedral was built between 1075 and 1211 with baroque and gothic elements mixed with the romantic style, in order to house the tomb of the Apostle Santiago. Its façade is impressive and you can enter its interior through the Puerta de Platerías to visit its 16 chapels. In the back of the High Altar of the Cathedral is the image of the apostle Santiago, where many believers stop to embrace it.
+
+![The Cathedral of Santiago of Compostela](_static/images/santiago-de-compostela/cathedral-of-santiago-of-compostela.jpg)
+
+The jewel of this temple is the Pórtico de la Gloria, which has numerous religious figures and you can also be surprised by the huge 50-kilo censer, set in motion to swing it during religious celebrations.
+
```
Comment (jbampton) [inline doc/source/santiago_de_compostela.md:18]:
```suggestion

```

File: doc/source/santiago_de_compostela.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+![The Cathedral of Santiago of Compostela](_static/images/santiago-de-compostela/cathedral-of-santiago-of-compostela.jpg)
+
+The jewel of this temple is the Pórtico de la Gloria, which has numerous religious figures and you can also be surprised by the huge 50-kilo censer, set in motion to swing it during religious celebrations.
+ 
+## Obradoiro Square
+
+The Square Obradoiro, in addition to the cathedral, has several interesting places to see such as the Pazo de Raxoi, which is the current seat of the town hall, the old episcopal palace, the Pazo de Xelmirez, and the old college of the city, the Colexio de Saint Xerome. However, the building preferred by most is the hospital of the Catholic Monarchs, which was built in 1501 to give shelter to pilgrims on the Camino de Santiago, today it is a hotel known as the Parador de Santiago and its interior is a true wonder.
+
+![Obradoiro Square](_static/images/santiago-de-compostela/obradoiro-square.jpg)
+
```
Comment (jbampton) [inline doc/source/santiago_de_compostela.md:24]:
```suggestion

```

File: doc/source/santiago_de_compostela.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+## Food Market
+
+The Food Market is the second most visited site after the Cathedral. There you can see local people buying seafood, fish, vegetables, fruits, meats, cheeses, etc. but you can also consume fresh products from the market such as oysters at A Ostrería, at Pulperia Abastos try their exquisite octopus or they cook for you at the moment at Mariscomania what you have bought in the market, for approximately five euros per diner.
+
+## Galician People's Museum
+
+The Museo do pobo Galego offers you a walk through Galician culture through its history, it is located in the Santo Domingo de Bonaval convent. You will be able to hallucinate with its triple helicoidal staircase and the Church, where the remains of famous Galicians such as the poet Rosalía de Castro are found.
+
+![Triple Helicoidal Staircase](_static/images/santiago-de-compostela/triple-helicoidal-staircase.jpg)
+
```
Comment (jbampton) [inline doc/source/santiago_de_compostela.md:44]:
```suggestion

```

**Review by jbampton — CHANGES_REQUESTED** [review]
https://slurp--1910.org.readthedocs.build/en/1910/santiago-de-chile.html

Entry missing from menu since the filename is wrong

**Comment by jbampton** [comment]
Screen Shot 2022-10-06 at 8 36 49 pm

File: doc/source/santiago-de-compostela.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# 🇪🇸 Santiago de Compostela
+
+Santiago de Compostela is a city that is located in the northwest of Spain and is one of the main places of pilgrimage of Christianity. This city falls in love at first sight with the beautiful streets of the old town, declared a World Heritage Site, for its monumental Cathedral and its prestigious university that is more than 500 years old.
+![Santiago de Compostela cityscape](_static/images/santiago-de-compostela/cityscape.jpg)
```
Comment (jbampton) [inline doc/source/santiago-de-compostela.md:4]:
```suggestion

![Santiago de Compostela cityscape](_static/images/santiago-de-compostela/cityscape.jpg)
```

File: doc/source/santiago-de-compostela.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+The Alameda Park offers you walks among eucalyptus, chestnut and oak trees as well as several statues such as that of Rosalía de Castro, that of the two Marías and that of the playwright Valle Inclán. You can also go to the viewpoints on the Paseo de la Herradura where you will get fantastic views of the historic center and the Cathedral.
+
+## River walk along the Sarela River
+
+Bordering the Sarela River, you can take a pleasant walk, starting with the Iglesia del Carmen de Abaixo.
+
+Another worthwhile route is the Sar river path with its beautiful natural landscapes and you will be able to see a Romanesque temple such as the Colexiata de Santa María do Sar, built in the 12th century where you will be impressed by its interior and the inclination of its columns. Near this area there is an architectural complex designed by the architect Peter Eisenman, which is the City of Culture of Galicia, you will be pleasantly surprised by its design and its forms, in addition to its unique buildings offering an extensive cultural offer, which you cannot Stop watching.
+
+![City of Culture of Galicia ](_static/images/santiago-de-compostela/city-of-culture-galicia.jpg)
```
Comment (jbampton) [inline doc/source/santiago-de-compostela.md:62]:
```suggestion
![City of Culture of Galicia](_static/images/santiago-de-compostela/city-of-culture-galicia.jpg)
```

**Pull request merged**